### PR TITLE
Scaffolded out the profile component to show the same values as the e…

### PR DESCRIPTION
**Pull request by robstetson**
## Description

After working on the Edit Profile modal we realized there was static information on the update profile page. Implemented the proper fields to go alongside the Edit profile modal. Simple scaffolding of that page to get it in line with the other components.

#### Video Link

[Loom Video] https://www.loom.com/share/26738496f43a4aa9a109f8f86c8fb2a0

#### Trello Link

https://trello.com/c/dPAK5bRb/295-as-a-user-i-want-to-see-the-my-profile-fields-on-the-edit-modal-rendered-on-the-my-profile-page

(Example below):

## Type of change

___Please delete options that are not relevant.___

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

## Contributors

@BryceSlade 

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by areccus — APPROVED** [review]

**Review by justinpeisker — APPROVED** [review]
This approach of incrementally stepping toward full fledged functionality is great.  You've definitely driven progress on the profile page.  Well done.  Now let's get it wired up!

File: src/components/pages/Profile/RenderProfileContainer.js [hunk 1]
```diff
@@ -66,6 +66,27 @@ const RenderProfileContainer = props => {
             <Card type="inner" title="Bio" extra={<EditOutlined />}>
               {userData.bio || 'Profile bio will go here.'}
             </Card>
+
```
Comment (ryan-hamblin) [inline src/components/pages/Profile/RenderProfileContainer.js:69]: I'd appreciate if you removed the line breaks between your components in the component tree. I mention this as a way to establish uniformity within the app. Some components are spaced out while some aren't in this file.

**Review by ryan-hamblin — CHANGES_REQUESTED** [review]
This is pretty straightforward. Good use of the AntD components. I suggest a little bit of clean up for uniformity. No need to rerecord your loom once you get the edits changed.

File: src/components/pages/Profile/RenderProfileContainer.js (hunk 1 above)
Comment (robstetson) [reply src/components/pages/Profile/RenderProfileContainer.js:69]: For sure! I will go ahead and get that taken care of asap!

**Review by wscottreynders — APPROVED** [review]
This looks great.

**Review by jonathan-mcgraw — APPROVED** [review]
Beautiful work as always, sir.

File: src/components/pages/Profile/RenderProfileContainer.js [hunk 2]
```diff
@@ -66,6 +66,21 @@ const RenderProfileContainer = props => {
             <Card type="inner" title="Bio" extra={<EditOutlined />}>
               {userData.bio || 'Profile bio will go here.'}
             </Card>
+            <Card type="inner" title="Full Name" extra={<EditOutlined />}>
+              {userData.full_name || 'Full Name will go here.'}
```
Comment (phillipgalbraith) [inline src/components/pages/Profile/RenderProfileContainer.js:70]: Why require the backend to send the full name?  We do often want to get a full name on the front end. Maybe we should use a utility that joins them in the front end.

**Review by phillipgalbraith — APPROVED** [review]
The description of this PR gives great justification for each type of data in the scaffolding.  Nice clean code.

**Review by arsalan2243 — APPROVED** [review]
this new added feature is pretty solid. the field rendering ability would make it easier to work with.

**Review by OrdainedRat — APPROVED** [review]
Looks good!

**Review by wscottreynders — APPROVED** [review]
Great addition!

**Pull request merged**